codefeedr-plugins/codefeedr-travis/src/main/scala/org/codefeedr/plugins/travis/TravisProtocol.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.codefeedr.plugins.travis
+
+import java.time.LocalDateTime
+import org.codefeedr.pipeline.PipelineItem
+import org.codefeedr.plugins.github.GitHubProtocol.PushEvent
+
+object TravisProtocol {
+
+  case class PushEventFromActiveTravisRepo(pushEventItem: PushEvent) extends PipelineItem
+
+  case class TravisBuilds(`@type`: String,
+                          `@href`: String,
+                          `@representation`: String,
+                          `@pagination`: TravisPagination,
+                          builds: List[TravisBuild])
+
+  case class TravisBuild(`@type`: String,
+                         `@href`: String,
+                         `@representation`: String,
+                         `@permissions`: TravisPermissions,
+                         id: Int,
+                         number: String,
+                         state: String,
+                         duration: Option[Int],
+                         event_type: String,
+                         previous_state: String,
+                         pull_request_title: Option[String],
+                         pull_request_number: Option[Int],
+                         started_at: Option[LocalDateTime],
```

Comment:
Shouldn't it be updated to Date instead of LocalDateTime?

Reply:
The Travis plugin relies on the Github plugin which still uses LocalDateTime. I'll make an Issue to convert all LocalDateTimes in the framework to Date

Reply:
I already created an issue for both GH and Travis here: #90